CONTRIBUTING.md

```diff
@@ -78,6 +78,21 @@ The following GitHub users are the currently available sponsors, either by being
 * [@djaglowski](https://github.com/djaglowski)
 
 Whenever a sponsor is picked from the top of this list, please move them to the bottom.
+
+## Adding metrics to existing receivers
+Following these steps for contributing additional metrics to existing receivers.
+ - Make sure you have basic tools, like `golang, make and git` installed in your system. 
+ - Fork this repo and check it out locally to your system, e.g. into the directory `opentelemetry-collector-contrib`
+ - Make sure $GOPATH is added to $PATH, if not sure, run `export PATH=$PATH:$(go env GOPATH)/bin`
+ - Run `cd opentelemetry-collector-contrib`
+ - Run `make install-tools`
```

Comment:
Should we link to these instructions instead? https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#fork I believe they cover the same thing; if something is missing we should add it to that document

Reply:
Did notice there was the similar part in that repository. I will update this doc with links then.